spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/batch/SimpleJobService.java

```diff
@@ -242,10 +242,12 @@ public JobExecution stop(Long jobExecutionId) throws NoSuchJobExecutionException
 		if (!jobExecution.isRunning()) {
 			throw new JobExecutionNotRunningException("JobExecution is not running and therefore cannot be stopped");
 		}
-
+		// Indicate the execution should be stopped by setting it's status to
```

Comment:
This is the replacement for [step 1) from the PR description](https://github.com/spring-projects/spring-batch/blob/fb0ce98a890eae1908bdf25d69eb4ea51b89973e/spring-batch-core/src/main/java/org/springframework/batch/core/jsr/launch/JsrJobOperator.java#L733-L742), correct?